app/models.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
```

Comment:
`sqlite3.connect("cinema.sqlite") ` should work without building a relative path